subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -1050,7 +1050,6 @@ static int lwm2m_send_message(struct lwm2m_message *msg)
 	if (rc < 0) {
 		if (errno == EAGAIN || errno == EWOULDBLOCK) {
 			LOG_WRN("Failed to send packet, would block");
-			return -errno;
```

Comment:
In that case, we should perhaps drop the entire `if (errno == EAGAIN || errno == EWOULDBLOCK) {` condition? The error will be logged just below.

Reply:
Good point, I removed this condition